src/test/java/dk/cngroup/lentils/api/ContactRestControllerTest.kt

```diff
@@ -0,0 +1,31 @@
+package dk.cngroup.lentils.api
+
+import dk.cngroup.lentils.dto.ContactFormDTO
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.boot.test.web.client.TestRestTemplate
+import org.springframework.http.HttpStatus
+import org.springframework.test.context.junit4.SpringRunner
+
+@RunWith(SpringRunner::class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+class ContactRestControllerTest {
+
+    @Autowired
+    lateinit var testRestTemplate: TestRestTemplate
+
+    @Test
+    fun contactUrl() {
```

Comment:
Tyto testy jsou napsane zatim jen pro to, jestli je api definovane nebo neni (tj. zda je definovany handler pro urcite url). V tom pripade bych jeste poprosil otestovat take POST request na /api/contact a zaroven bych prejmenoval metody:
contactUrl -> napr.: testValidContactUrl, validContactUrlTest (nebo jen validContactUrl), pokdu rekneme, ze v kotlinu nebudeme chtit slovo "test" jako prefix nebo postfix.
a nonsenseUrl -> testInvalidContactUrl, invalidContactUrlTest nebo jen (invalidContactUrl) -||-

Reply:
Zaroven by se pak hodily i testy na to, zda se vrati spravny json z toho api.